### DISCUSSION TITLE
Dynamic metadata for every chunk while upserting a document

### DISCUSSION THREAD

**Discussion by prithvi151080** [FlowiseAI/Flowise#2239] [General]
Hi, has anyone tried to create dynamic metadata for every chunk while upserting a document? I am trying to create a RAPTOR instance for upserting documents in the vector DB  where in every chunk will have a summary metadata that summarizes the semantic meaning of the chunk. Any help will be highly appreciated.

**Comment by prithvi151080** [comment]
Am i the only one trying to create a RAPTOR RAG???

**Reply by mikehudson2** [reply]
Same

**Reply by pgaurav007** [reply]
Did you guys try it further? I am also having similar usecase.
